pkg/cmd/pulumi/doc.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016-2020, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main defines all of the commands for the Pulumi CLI, and also includes logic related to the CLI lifecycle
```

Comment:
I don't think that this needs to be `Package main...` b/c it is documenting an executable.

Reply:
Not sure I understand -- are you suggesting to drop this comment entirely?

Reply:
No, just the `Package main` bit. Instead, we should simply document that this is the implementation of the Pulumi CLI. See e.g. the docs for the `go` tool: https://golang.org/src/cmd/go/alldocs.go